mscore/musescore.cpp

```diff
@@ -7103,9 +7103,10 @@ bool MuseScore::saveMp3(Score* score, QIODevice* device, bool& wasCanceled)
 
 void tryToRequestTelemetryPermission()
       {
+      static const QString telemetryVersion = "3.4.0";
       QString accessRequestedAtVersion = preferences.getString(PREF_APP_STARTUP_TELEMETRY_ACCESS_REQUESTED);
 
-      if (accessRequestedAtVersion == VERSION)
+      if (accessRequestedAtVersion == telemetryVersion)
```

Comment:
I believe every version change, which really is a code change, potentially involving telemetry data too, requires to ask permission again.

Reply:
It seems to me it doesn't make sense to ask for this permission again if no really massive changes to telemetry data collection happen and if no Privacy Policy changes occur. But if necessary I can move this change to a separate pull request to discuss it in more details, it doesn't make any difference before we release 3.4.1 version anyway.

Reply:
> I believe every version change, which really is a code change, potentially involving telemetry data too, requires to ask permission again.

Why? We don't specify specific data we are going to collect in the Telemetry dialog. If users agree to send anonymous data, it will be kept in future updates, too.

Reply:
this now is a fixed string, so as long as that doiesn't get changed any other telemerty code change won't be taken into re-consideration by the users.
If it were to check for major and minor, but not for patch level this might be acceptable, the way it is now is not.

Reply:
Removed the relevant commit for now, will open a separate PR soon.

Reply:
See #5618.